src/ngraph_encapsulate_op.cc

```diff
@@ -588,6 +587,7 @@ class NGraphEncapsulateOp : public OpKernel {
                         "Caught exception while executing nGraph computation: ",
                         exp.what(), "\n"));
       } catch (...) {
+        ng_function = m_ng_function_map[ng_exec];
```

Comment:
This is a fix to my earlier PR, without this line it would not dump serialize graph if backend compile call fails.